diff -up chrony-3.2/examples/chrony.conf.example2.defconfig chrony-3.2/examples/chrony.conf.example2
--- chrony-3.2/examples/chrony.conf.example2.defconfig	2017-09-15 08:32:09.000000000 +0200
+++ chrony-3.2/examples/chrony.conf.example2	2017-09-18 12:48:08.688020942 +0200
@@ -28,9 +28,6 @@ rtcsync
 # Specify file containing keys for NTP authentication.
 #keyfile /etc/chrony.keys
 
-# Get TAI-UTC offset and leap seconds from the system tz database.
-#leapsectz right/UTC
-
 # Specify directory for log files.
 logdir /var/log/chrony
 
--- chrony-3.4/examples/chrony.keys.example	2018-09-19 16:38:15.000000000 +0200
+++ chrony-3.4/examples/chrony.keys.example	2017-09-15 08:32:09.000000000 +0200
@@ -1,7 +1,6 @@
-# This is an example chrony keys file.  It enables authentication of NTP
-# packets with symmetric keys when its location is specified by the keyfile
-# directive in chrony.conf(5).  It should be readable only by root and the
-# user under which chronyd is running.
+# This is an example chrony keys file.  It is used for NTP authentication with
+# symmetric keys.  It should be readable only by root or the user to which
+# chronyd is configured to switch to after start.
 #
 # Don't use the example keys!  It's recommended to generate random keys using
 # the chronyc keygen command.
