diff -up chrony-3.2/doc/chrony.conf.man.in.timestamping chrony-3.2/doc/chrony.conf.man.in
--- chrony-3.2/doc/chrony.conf.man.in.timestamping	2017-09-15 10:17:41.000000000 +0200
+++ chrony-3.2/doc/chrony.conf.man.in	2017-09-19 13:52:59.544187046 +0200
@@ -3201,13 +3201,12 @@ timestamping. If the server or peer supp
 be enabled by the \fBxleave\fP option in the \fBserver\fP or the
 \fBpeer\fP directive.
 .sp
-This directive is supported on Linux 3.19 and newer. The NIC must support HW
+This directive is supported on Linux. The NIC must support HW
 timestamping, which can be verified with the \fBethtool \-T\fP command. The list of
 capabilities should include \fISOF_TIMESTAMPING_RAW_HARDWARE\fP,
 \fISOF_TIMESTAMPING_TX_HARDWARE\fP, and \fISOF_TIMESTAMPING_RX_HARDWARE\fP. Receive
 filter \fIHWTSTAMP_FILTER_ALL\fP, or \fIHWTSTAMP_FILTER_NTP_ALL\fP, is necessary for
-timestamping of received packets. Timestamping of packets received from bridged
-and bonded interfaces is supported on Linux 4.13 and newer. When \fBchronyd\fP is
+timestamping of received packets. When \fBchronyd\fP is
 running, no other process (e.g. a PTP daemon) should be working with the NIC
 clock.
 .sp
